app/Models/Scopes/OnlineScope.php

```diff
@@ -11,7 +10,7 @@ class OnlineScope implements Scope
 {
     public function apply(Builder $builder, Model $model)
     {
-        if ($model instanceof Draftable) {
+        if (class_has_trait(get_class($model), 'App\Models\Traits\Draftable')) {
```

Comment:
I'm thinking you can use the `::class` syntax. Search the Laravel source code for usages of `class_uses_recursive` to see some examples.

Reply:
Since we have control over the `class_has_trait`, maybe allow us to pass an object too, so we can drop the `get_class` call where it's used?

Then we'd en up with `class_has_trait($model, Draftable::class)`

Maybe you should look into PR'ing an overloaded version of `class_uses_recursive` that does exactly this before 5.6, imo would clean up the Laravel code base too.